controllers/apply/awards-for-all/__snapshots__/form.test.js.snap

```diff
@@ -30,7 +30,6 @@ Array [
   "seniorContactDateOfBirth: Enter a date of birth",
   "seniorContactAddress: Enter a full UK address",
   "seniorContactAddressHistory: Enter a full UK address",
-  "seniorContactEmail: Enter an email address",
```

Comment:
Not entirely sure why this changed, but probably fine...

Reply:
Dug into this and made a small change, added a `.required()` to each. This is not _strictly_ required as the .email() validation acts as a requirement as far as form submissions are concerned but it does make the snapshot tests clearer and is a belt and braces check.